types/vimeo__player/index.d.ts

```diff
@@ -86,6 +87,9 @@ export interface ChapterChangeEvent {
     index: number;
 }
 
+// tslint:disable-next-line:no-empty-interface
+export interface ChapterChangeEvent extends VimeoChapter {}
```

Comment:
I think using `VimeoChapter` directly instead of `ChapterChangeEvent` would be better

Reply:
Thanks Sheetal... I've made the changes as suggested 👋